Doxyfile

```diff
@@ -811,7 +811,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                =
+EXCLUDE                = travisci/ tools/ tests/
```

Comment:
As long as we are explicitly including files in INPUT does EXCLUDE really have any effect?